### PR TITLE
Fix script names and gitignore

### PR DESCRIPTION
**Pull request by samarmohan**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -4,19 +4,19 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn run build:api && yarn workspace @oasis/web run build",
+    "build": "yarn workspace @oasis/web run build",
     "build:api": "yarn workspace @oasis/api run build && yarn workspace @oasis/client-gql run build",
     "clean": "yarn workspaces foreach --include @oasis/api,@oasis/client-gql,@oasis/web -p run clean",
     "commit": "git add . && cz && git push",
-    "deploy": "sh bin/deploy.sh",
+    "deploy:dev": "sh bin/deploy.sh",
```
Comment (heybereket) [inline package.json:11]: Keep this as "deploy" because it's deploying to prod, maybe try deploy:prod?

File: package.json [hunk 2]
```diff
@@ -4,19 +4,19 @@
     "packages/*"
   ],
   "scripts": {
-    "build": "yarn run build:api && yarn workspace @oasis/web run build",
+    "build": "yarn workspace @oasis/web run build",
```
Comment (heybereket) [inline package.json:7]: Keep the build command as it is, "yarn build" is supposed to build both the API and Web, but people can specify what to build (ex. yarn build:api)

File: package.json [hunk 3]
```diff
@@ -4,19 +4,19 @@
...
     "build:api": "yarn workspace @oasis/api run build && yarn workspace @oasis/client-gql run build",
     "clean": "yarn workspaces foreach --include @oasis/api,@oasis/client-gql,@oasis/web -p run clean",
     "commit": "git add . && cz && git push",
-    "deploy": "sh bin/deploy.sh",
+    "deploy:dev": "sh bin/deploy.sh",
     "deploy:prod": "yarn lint && git add . && git commit -m \"chore(deploy): Merge staging -> prod\" &&git pull origin staging  && git checkout prod && git merge staging && git push origin prod && git checkout staging",
-    "dev": "yarn workspace @oasis/web dev",
+    "dev:web": "yarn workspace @oasis/web dev",
```
Comment (heybereket) [inline package.json:13]: Not needed, dev starts the dev server, just to keep it simple it can stay as dev.

File: package.json [hunk 4]
```diff
@@ -4,19 +4,19 @@
...
     "dev:api": "yarn workspaces foreach --include @oasis/client-gql,@oasis/api -p run dev",
     "lint": "yarn workspaces foreach --exclude @oasis/client-gql -p run lint && yarn workspaces foreach --exclude @oasis/client-gql -p run prettify",
     "prepare": "husky install",
     "publish:bot": "sh bin/publish.sh",
     "reinstall": "yarn clean && rimraf packages/**/*.sqlite || yarn install",
-    "start": "yarn workspace @oasis/web run start"
+    "start:web": "yarn workspace @oasis/web run start"
```
Comment (heybereket) [inline package.json:19]: As I said for "yarn dev", "yarn start:web" isn't needed.

**Review by heybereket — COMMENTED** [review]
Thanks for the pull request, however, I'll have to close this PR as I don't think it's made any changes to improve Oasis as we want to make scripts and folders necessary and as simple for contributors and everyone else.

**Comment by HenryLeC** [comment]
What is this pr supposed to do or add?

**Pull request closed without merging**